docs/V1/Partner/respondents.md

```diff
@@ -68,9 +74,34 @@ author    | Prénom et Nom de l'auteur du commentaire
 author_id | Id de l'auteur du commentaire
 comment   | Commentaire
 
+## Récupérer un répondant d'un questionnaire
+
+### Requête
+**GET** `api-saas/test/v1/partner/questionnaires/{slug}/respondents/{respondentId}.json`
+
+### Réponse
+`200` - Les données d'un candidat du questionnaire au format JSON
+
+Voici les données retournées :
+
+data             | description
+-----------------|------------------
+id               | Id du répondant 
+score_auto       | Scoring automatique
+score_total      | Note total
+phone_number     | Numéro de téléphone du répondant
+main_comment     | Commentaire principal du répondant
+email            | Email du répondant 
+first_name       | Prénom du répondant 
+last_name        | Nom du répondant 
+candidate_link   | Lien vers la fiche du répondant (utilisateur connecté sur Scoringline) 
+synthesis_link   | Lien vers la synthèse PDF du répondant (utilisateur connecté sur Scoringline) 
+comments         | Tableau de commentaires concernant le répondant
+documents        | L'ensemble des documents (CV, synthèse) encodés au format base64
```

Comment:
@tontonjulien I indeed thought about that but it wouldn't give much information since it only returns the base64encoded file. There is no extra information.